package/MDAnalysis/analysis/rdf.py

```diff
@@ -261,6 +273,13 @@ def _single_frame(self):
             mask = np.where(idxA != idxB)[0]
             dist = dist[mask]
 
+        if self.exclude_same:
```

Comment:
```suggestion
        if self.exclude_same is not None:
```
`get()` can return `None`, maybe an explicit check is better because narrower?